packages/react-vis/src/animation.js

```diff
@@ -46,7 +46,7 @@ export function extractAnimatedPropValues(props) {
   const {animatedProps, ...otherProps} = props;
 
   return animatedProps.reduce((result, animatedPropName) => {
-    if (otherProps.hasOwnProperty(animatedPropName)) {
+    if (Object.prototype.hasOwnProperty.call(otherProps, animatedPropName)) {
```

Comment:
Is that in case `otherProps` is not an object?

Reply:
This was brought up by a line rule [no-prototype-builtins](https://eslint.org/docs/rules/no-prototype-builtins) to protect against the case that `otherProps` (or `props` itself) doesn't have its prototype set.
This can be done with `Object.create(null)`